docs/source/tutorial/spaceflights_tutorial.md

```diff
@@ -1,39 +1,32 @@
-# Kedro spaceflights tutorial
+# Introducing spaceflights
 
 In this tutorial, we construct nodes and pipelines for a price-prediction model to illustrate the steps of a typical Kedro workflow.
 
-In the text, we assume you have started with an empty Kedro project and we show the steps necessary to convert it into a working project. The tutorial guides you to copy and paste example code into the Kedro project. It takes approximately **one hour** to complete.
+The tutorial takes approximately **30 minutes** to complete. You will work in the terminal and by inspecting project files in an IDE or text editor. There is no Jupyter notebook for the project.
 
-```{note}
-You may prefer to get up and running more swiftly. We also provide the example as a [Kedro starter](../kedro_project_setup/starters.md) so you can follow along without copy/pasting.
-```
 
 ## Scenario
 
 *It is 2160, and the space tourism industry is booming. Globally, thousands of space shuttle companies take tourists to the Moon and back. You have been able to source data that lists the amenities offered in each space shuttle, customer reviews, and company information.*
 
 ***Project***: *You want to construct a model that predicts the price for each trip to the Moon and the corresponding return flight.*
 
-
 ![](../meta/images/moon-rocket.png)
 
 Photo by <a href="https://unsplash.com/@ivvndiaz">Ivan Diaz</a> on <a href="https://unsplash.com/s/photos/spaceship">Unsplash</a>
 
 
 ## Get help
-If you hit an issue with the tutorial, the Kedro community can help!
-
-Things you can do:
+If you hit an issue with the tutorial:
 
-* check the [spaceflights tutorial FAQ](spaceflights_tutorial_faqs.md) to see if we have answered the question already
-* use the spaceflights starter to [create a new, separate project which contains working example code](./tutorial_template.md#create-a-new-project), and compare that project with your own
-* use [Kedro-Viz](../visualisation/kedro-viz_visualisation) to visualise your project to better understand how the datasets, nodes and pipelines fit together
-* use the [#questions channel](https://slack.kedro.org/) on our Slack channel (which replaces our Discord server) to ask the community for help
-* search the [searchable archive of Discord discussions](https://linen-discord.kedro.org/)
+* Check the [spaceflights tutorial FAQ](spaceflights_tutorial_faqs.md) to see if we have answered the question already.
+* Use [Kedro-Viz](../visualisation/kedro-viz_visualisation) to visualise your project to better understand how the datasets, nodes and pipelines fit together.
+* Use the [#questions channel](https://slack.kedro.org/) on our Slack channel to ask the community for help.
+* Search the [searchable archive of Discord discussions](https://linen-discord.kedro.org/).
```

Comment:
Should we now refer folks to the newer Slack archive? The Discord archive does have historical value of course, but we've used Slack for enough time now that we've accumulated a sufficiently large knowledge base perhaps:

```suggestion
* Search the [searchable archive of Slack discussions](https://www.linen.dev/s/kedro).
```

(by the way, are we thinking of adding a `linen-slack.kedro.org`? @yetudada @tynandebold